modules/administration-guide/partials/proc_rolling-the-upgrade-back-to-che-7-41-on-openshift.adoc

```diff
@@ -0,0 +1,51 @@
+:parent-context-of-upgrading-che-7-41-on-openshift: {context}
+
+[id="rolling-the-upgrade-back-to-{prod-prev-id-short}-{prod-last-version-pre-dwo}-on-openshift_{context}"]
+= Rolling the upgrade back to {prod-prev-short} {prod-last-version-pre-dwo} on OpenShift
+
+If upgrading to {prod-short} {prod-ver} on OpenShift fails, you can run a rollback script to restore {prod-last-version-pre-dwo}.
+
+.Procedure
+
+. Download the xref:attachment$migration/rollback.sh[rollback.sh] script.
+
+. Set the environment variables to use in the `rollback.sh` script:
++
+[source,bash,subs="+attributes"]
+----
+export INSTALLATION_NAMESPACE={prod-namespace} # <1>
+export PRODUCT_ID={prod-id}
+export PRODUCT_SHORT_ID={prod-id-short}
+export PRODUCT_DEPLOYMENT_NAME={prod-deployment}
+export PRE_MIGRATION_PRODUCT_OPERATOR_NAMESPACE={prod-namespace} # <2>
+export PRE_MIGRATION_PRODUCT_DEPLOYMENT_NAME={pre-migration-prod-deployment}
+export PRE_MIGRATION_PRODUCT_SUBSCRIPTION_NAME={pre-migration-prod-subscription}
+export PRE_MIGRATION_PRODUCT_CHE_CLUSTER_CR_NAME={pre-migration-prod-checluster}
+export PRE_MIGRATION_PRODUCT_OPERATOR_NAME={pre-migration-prod-operator}
+export PRE_MIGRATION_PRODUCT_OLM_PACKAGE={pre-migration-prod-package}
+export PRE_MIGRATION_PRODUCT_OLM_CHANNEL={pre-migration-prod-channel}
+export PRE_MIGRATION_PRODUCT_OLM_CATALOG_SOURCE={pre-migration-prod-catalog-source}
+export PRE_MIGRATION_PRODUCT_OLM_STARTING_CSV={pre-migration-prod-starting-csv}
+----
+<1> `{prod-namespace}` or another {orch-namespace} where {prod-prev-short} was previously installed.
+<2> `{prod-namespace}` or another {orch-namespace} where {prod-prev-short} was previously installed.
+
+. Run `chmod` on the `rollback.sh` script:
++
+[source,terminal]
+----
+$ chmod +x ./rollback.sh
```

Comment:
personally, I'd combine these two steps into a single command, because anyone running linux-based shell commands doesn't need to be told to run chmod (and the instructions don't explain what chmod is, only that you should run it ... so you're not educating anyone on WHY the file needs to be executable).

Thus:

```
. Run the `rollback.sh` script.
+
[source,terminal]
----
$ chmod +x ./rollback.sh; ./rollback.sh
----
```

Reply:
Thank you, Nick. I've applied this suggestion to both procedures now in a new commit.